apps/frontend/src/pages/auth/libs/components/sign-up-form/sign-up-form.tsx

```diff
@@ -1,3 +1,5 @@
+import { UserValidationRule } from "shared/src/modules/users/libs/enums/enums.js";
```

Comment:
Same comment regarding UserValidationRule as it was descibed for `sign-in-form.tsx`